assets/images/game-ci-brand-logo-wordmark.svg

```diff
@@ -1,2 +1,119 @@
+      <path
+         d="M 148.66152,706.166 H 411.93095 V 531.22506 H 148.66152 Z m 279.42159,32.30119 H 132.51247 c -8.92011,0 -16.15216,-7.23205 -16.15216,-16.15215 V 515.0729 c 0,-8.9169 7.23205,-16.14895 16.15216,-16.14895 h 295.57064 c 8.9201,0 16.15215,7.23205 16.15215,16.14895 v 207.24214 c 0,8.9201 -7.23205,16.15215 -16.15215,16.15215"
+         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:3.13755"
+         id="path106" />
+      <path
+         d="m 348.24514,792.82466 h -135.8972 c -8.92002,0 -16.14895,-7.23205 -16.14895,-16.14895 0,-8.92012 7.22893,-16.15536 16.14895,-16.15536 h 135.8972 c 8.92002,0 16.15207,7.23524 16.15207,16.15536 0,8.9169 -7.23205,16.14895 -16.15207,16.14895"
+         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:3.13755"
+         id="path110" />
+      <path
+         d="m 692.28577,788.09082 h -98.34371 c -33.59073,0 -58.51233,-6.71435 -74.7649,-20.15254 -16.26201,-13.43187 -24.38513,-34.12098 -24.38513,-62.06722 v -125.3486 c 0,-27.94624 8.12312,-48.70437 24.38513,-62.27115 16.25257,-13.5637 41.17417,-20.35657 74.7649,-20.35657 h 98.34371 c 5.1016,0 8.79773,1.14838 11.08186,3.42938 2.28101,2.28413 3.42929,5.98016 3.42929,11.08187 v 48.77024 c 0,5.10483 -1.14828,8.80086 -3.42929,11.08497 -2.28413,2.28101 -5.98026,3.42307 -11.08186,3.42307 h -81.01178 c -8.33338,0 -14.50813,1.68175 -18.5399,5.03581 -4.03488,3.36659 -6.04605,8.40237 -6.04605,15.11682 v 95.92457 c 0,6.72066 2.01117,11.62465 6.04605,14.71207 4.03177,3.09355 10.20652,4.63415 18.5399,4.63415 h 81.01178 c 5.1016,0 8.79773,1.14829 11.08186,3.42929 2.28101,2.28422 3.42929,5.98025 3.42929,11.08195 v 47.96385 c 0,5.10481 -1.14828,8.80394 -3.42929,11.07872 -2.28413,2.29036 -5.98026,3.42932 -11.08186,3.42932"
+         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:3.13755"
+         id="path114" />
+      <path
+         d="m 760.39904,788.09082 c -5.10481,0 -8.80707,-1.13896 -11.08497,-3.42932 -2.28733,-2.27478 -3.42308,-5.97391 -3.42308,-11.07872 v -47.96385 c 0,-5.1017 1.13575,-8.79773 3.42308,-11.08195 2.2779,-2.281 5.98016,-3.42929 11.08497,-3.42929 h 39.90039 V 575.68427 h -39.90039 c -5.10481,0 -8.80707,-1.14206 -11.08497,-3.42307 -2.28733,-2.28411 -3.42308,-5.98014 -3.42308,-11.08497 v -48.77024 c 0,-5.10171 1.13575,-8.79774 3.42308,-11.08187 2.2779,-2.281 5.98016,-3.42938 11.08497,-3.42938 h 171.69981 c 5.10171,0 8.79462,1.14838 11.08507,3.42938 2.2747,2.28413 3.42307,5.98016 3.42307,11.08187 v 48.77024 c 0,5.10483 -1.14837,8.80086 -3.42307,11.08497 -2.29045,2.28101 -5.98336,3.42307 -11.08507,3.42307 h -39.09397 v 135.42342 h 39.09397 c 5.10171,0 8.79462,1.14829 11.08507,3.42929 2.2747,2.28422 3.42307,5.98025 3.42307,11.08195 v 47.96385 c 0,5.10481 -1.14837,8.80394 -3.42307,11.07872 -2.29045,2.29036 -5.98336,3.42932 -11.08507,3.42932 z"
+         style="fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:3.13755"
```

Comment:
For dark-reader we might force fill color to white to prevent this 
![image](https://user-images.githubusercontent.com/20756439/116812255-261a7900-ab4e-11eb-8b88-f79057a6531a.png). Or it could be based on `color` property, so in the menu it could use the text color of the menu items and on a white background it could use a darker color.

Not sure what's an elegant way to do it - we might also just duplicate and hardcode it for dark and light backgrounds for now.

Reply:
Indeed I don't exactly know how to achieve this, but I'll open an issue and a different branch to test this. I wouldn't wait for merging this as it's better anyway, but nice catch! :D